### PR TITLE
feat: Added automated checkout notebook

### PR DESCRIPTION
**Pull request by antoniomtz**
## PR Checklist

- [x ] Added label to the Pull Request for easier discoverability and search
- [ x] Commit Message meets guidelines as indicated in the URL https://github.com/intel-retail/automated-self-checkout/blob/main/CONTRIBUTING.md
- [ ] Every commit is a single defect fix and does not mix feature addition or changes
- [ ] Unit Tests have been added for new changes
- [ ] Updated Documentation as relevant to the changes
- [ ] All commented code has been removed
- [ ] If you've added a dependency, you've ensured license is compatible with repository license and clearly outlined the added dependency.
- [ ] PR change contains code related to security
- [ ] PR introduces changes that breaks compatibility with other modules (If YES, please provide details below)

## What are you changing?

Added jupyter notebook for automated checkout use case

## Issue this PR will close

close: #**issue_number**

## Anything the reviewer should know when reviewing this PR?

1. cd notebooks
2. docker build . -t automated-checkout
3. docker run -it --device=/dev/dri --device=/dev/video0 --privileged --group-add=$(stat -c "%g" /dev/dri/render* | head -n 1) -p 8888:8888 automated-checkout
4. open URL that was prompted in console: http://localhost:8888/lab?token=<token>
5. Run notebook

### CODE REVIEW

File: notebooks/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#
+# Copyright (C) 2023 Intel Corporation.
```
Comment (jim-wang-intel) [inline notebooks/Dockerfile:2]:
```suggestion
# Copyright (C) 2024 Intel Corporation.
```

**Review by jim-wang-intel — APPROVED** [review]
LGTM

File: notebooks/automated-checkout.ipynb [hunk 2]
```diff
@@ -0,0 +1,950 @@
...
+    "- Live demo"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "d7a12678-b12f-48d1-9735-398855733e46",
+   "metadata": {},
+   "source": [
+    "## Get PyTorch model\n",
```
Comment (NeethuES-intel) [inline notebooks/automated-checkout.ipynb:26]: so this is pytorch based and not tensorflow ?

File: notebooks/README.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
+# Automated checkout using yolov8, text detection and recognition models with OpenVINO™
+
+This Jupyter notebook demonstrates an automated checkout system that integrates object detection and text detection/recognition AI models. Utilizing YOLOv8 for object detection, the notebook efficiently identifies various items within a checkout frame, leveraging its robust and precise detection capabilities. To complement this, the notebook incorporates text detection and recognition within designated regions of interest. This combination allows for the extraction and interpretation of textual information—such as product labels and prices—from the detected items, facilitating a seamless and automated checkout process. The notebook is designed with detailed code explanations and visual outputs, making it accessible for users to understand and adapt the technology for various retail environments.
+
```
Comment (NeethuES-intel) [inline notebooks/README.md:4]: a pipeline architecture diagram here would be helpful for reader

**Review by NeethuES-intel — APPROVED** [review]
LGTM, few comments

File: notebooks/automated-checkout.ipynb (hunk 2 above)
Comment (antoniomtz) [reply notebooks/automated-checkout.ipynb:26]: yes, yolov8 is pytorch based

File: notebooks/README.md (hunk 3 above)
Comment (antoniomtz) [reply notebooks/README.md:4]: I will add a diagram and gif. Potentially a video later on.
